prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -631,6 +631,8 @@ void rxr_pkt_handle_data_copied(struct rxr_ep *ep,
 	struct rxr_rx_entry *rx_entry;
 	ssize_t ret;
 
+	rxr_pkt_entry_release_rx(ep, pkt_entry);
+
 	rx_entry = pkt_entry->x_entry;
```

Comment:
I think this needs happen before releasing pkt_entry.

Reply:
Good catch! fixed in new revision.